tests/foreman/ui/test_activationkey.py

```diff
@@ -965,7 +965,7 @@ def test_negative_usage_limit(session, module_org, rhel6_contenthost):
 @pytest.mark.skip_if_not_set('clients')
 @pytest.mark.tier3
 @pytest.mark.upgrade
-@pytest.mark.skipif((not settings.repos_hosting_url), reason='Missing repos_hosting_url')
+@pytest.mark.skipif((not settings.robottelo.repos_hosting_url), reason='Missing repos_hosting_url')
```

Comment:
Another instance case sensitivity.  Not important to PR, but leaving this here.

Reply:
thanks. ultimately, I think that these should all be lowercase (since constant is assumed), but it isn't something I am currently enforcing.